.github/workflows/master.yml

```diff
@@ -79,17 +78,12 @@ jobs:
           java-version: ${{ matrix.java }}
           cache: 'maven'
           check-latest: false
-      - name: Build and test with maven w/o linters
-        run: >-
-          ./build/mvn clean install
-          -Dmaven.javadoc.skip=true
-          -Drat.skip=true
-          -Dscalastyle.skip=true
-          -Dspotless.check.skip
-          -Dorg.slf4j.simpleLogger.defaultLogLevel=warn
-          -Pspark-${{ matrix.spark }}
-          ${{ matrix.spark-archive }}
-          ${{ matrix.exclude-tags }}
+      - name: Build and test Kyuubi and Spark with maven w/o linters
+        run: |
+          MVN_OPT="-Dmaven.javadoc.skip=true -Drat.skip=true -Dscalastyle.skip=true -Dspotless.check.skip -Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
```

Comment:
we can define MVN_OPT env in the flow level.

Reply:
updated